docs/features/ui-language.md

```diff
@@ -37,19 +37,49 @@ If you build CKEditor from scratch or integrate it directly into your applicatio
 	// Define webpack plugins ...
 		plugins: [
 			new CKEditorWebpackPlugin( {
-				// Note: The plugin is currently able to build just one language at a time.
-				languages: [ 'pl' ]
+				// Main language that will be built in to the main bundle.
+				language: 'en',
+
+				// Additional languages that will be emitted to the `outputDirectory`.
+				// This option can be set to array of language codes or `'all'` to build all found languages.
+				// The bundle is optimized for one language when this option is omitted.
+				additionalLanguages: 'all',
+
+				// Optional directory for emitted translations, `'lang'` by default.
+				// Relative to the webpack's output.
+				outputDirectory: 'lang',
+
+				// Optional flag indicates breaking the build process when the error is found.
+				throwErrorOnMissingTranslation: false
```

Comment:
Is `false` a default value? In documentation, in code samples, we should present real usage and no one will change `false` to `false` :P Plus, the comment above is unclear now – if you'd turn it on here, we could simply document that it "Will cause the building process to fail if a translation for one of the strings is missing".

Reply:
Ok. I just wanted to list available options with their default values.